packages/api-plugin/src/generators/trpc-api-generator/files/package.json

```diff
@@ -10,7 +10,7 @@
     "clean": "rm -rf build types tsconfig.tsbuildinfo",
     "typecheck": "tsc --noEmit --emitDeclarationOnly false",
     "typecheck:ci": "tsc --noEmit --emitDeclarationOnly false",
-    "dev": "tsup --watch --onSuccess 'pnpm kill && pnpm start'",
+    "dev": "tsup --watch --onSuccess 'node build/cmd/run.js'",
```

Comment:
in most cases I've noticed the kill command isn't necessary, and just adds startup time